frontend/src/store/modules/auth/actions.js

```diff
@@ -0,0 +1,30 @@
+export default {
+    login: (context, user) => {
```

Comment:
add folder "types" and define action const definitions there
```
export const LOGIN = "LOGIN";
```
and use them inside action definition and in components